components/apimgt/org.wso2.carbon.apimgt.micro.gateway/org.wso2.carbon.apimgt.micro.gateway.configurator/src/main/java/org/wso2/carbon/apimgt/micro/gateway/configurator/ConfigConstants.java

```diff
@@ -26,6 +26,7 @@
     public static final String CARBON_HOME = "carbon.home";
     public static final String REPOSITORY_DIR = "repository";
     public static final String CONF_DIR = "conf";
+    public static final String CLOUD_CONFIG_DIR = "wso2-cloud";
```

Comment:
Why are we binding this to wso2 cloud or cloud? AFAIK, micro gateway is a pattern and it should not bind with wso2 cloud.

Reply:
@pubudu538 This component is specific to cloud and this will not be used by the default APIM product. Hence we keep cloud specific code in this.